### PR TITLE
Add /gov page and contact form

### PR DESCRIPTION
**Pull request by steverydz**
## Done

- Add /gov page
- Add /gov contact form

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/gov
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the page matches [the copy doc](https://docs.google.com/document/d/1umM1XMEzqeXklPkyCjDr98Q-RvwCcNNXgwNwfLoynTo/edit#)
- Check that the form matches [the copy doc](https://docs.google.com/document/d/1B-2oVj3NU8PYvPWohVYKrYkNFCOUlHS3b31c4J5l_Gk/edit#heading=h.tk4y6g6el1tm) and works as expected

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2847, https://github.com/canonical-web-and-design/web-squad/issues/2848

### CODE REVIEW

**Comment by steverydz** [comment]
@meralonso there may still be copy changes but you can review the design

**Comment by meralonso** [comment]
Hey @steverydz, thanks, it's looking nice. Couple of things:
1. The shields seem a bit shrank from top and bottom. I don't see any weird thing in the [svg file](https://assets.ubuntu.com/v1/338124d1-shields-security.svg) but let me know if this is a design issue and I'll fix it.
2. I like the logos in one line as per my design, but if that's not possible I guess it's fine
Screenshot 2020-06-12 at 15 00 28

**Comment by steverydz** [comment]
> The shields seem a bit shrank from top and bottom.

@meralonso I'm not sure what you mean? I can't see any difference between the image in the page and the SVG on it's own. It's using the same proportions.

> I like the logos in one line as per my design, but if that's not possible I guess it's fine

Yeah the logos often don't fit on one line - it's down to the spacing in the inline images pattern.

**Comment by meralonso** [comment]
@steverydz I've made a screenshot - original illustration on the lfs and the one I see in your page on the rhs. I drew some red arrows, can you see the difference?
Screenshot 2020-06-12 at 15 59 05

Same about the logos, can we make it a bit smaller or they're always same size?

**Comment by steverydz** [comment]
@meralonso ok I think it's fixed now

File: templates/gov/index.html [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+          <div class="icon-container__icon">
+            <i class="p-icon--dark-tick"></i>
+          </div>
+          <div class="icon-container__text">
+            <a href="/legal/data-privacy">Read our data privacy policy&nbsp;&rsaquo;</a>.
+          </div>
+        </td>
+      </tr>
+    </table>
+    <p><a href="#">Learn more about our certifications&nbsp;&rsaquo;</a></p>
```
Comment (sowasred2012) [inline templates/gov/index.html:368]: Not sure what this is supposed to link to (nothing in the copy doc), at the moment it just takes you to the top of the page.

File: templates/gov/index.html [hunk 2]
```diff
@@ -0,0 +1,484 @@
...
+        </td>
+      </tr>
+      <tr>
+        <td><strong>GDPR Compliance tested</strong></td>
+        <td>
+          <div class="icon-container__icon">
+            <i class="p-icon--dark-tick"></i>
+          </div>
+          <div class="icon-container__text">
+            <a href="/legal/data-privacy">Read our data privacy policy&nbsp;&rsaquo;</a>.
```
Comment (sowasred2012) [inline templates/gov/index.html:363]: IMO it looks weird to have this link followed by a period, given that there is no non-link copy, and the link is also followed by a right angle quote/arrow symbol

**Pull request merged**